src/backends/DomRenderer.js

```diff
@@ -36,12 +36,35 @@ function setStyle(rule: HTMLElement|CSSStyleRule, prop: string, value: string):
   return true
 }
 
+function extractSelector(cssText: string, from: number = 0) {
+  return cssText.substr(from, cssText.indexOf('{') - 1)
+}
+
+const CSSRuleTypes = {
+  STYLE_RULE: 1,
+  KEYFRAMES_RULE: 7
+}
 
 /**
  * Get the selector.
  */
-function getSelector(rule: CSSStyleRule): string {
-  return rule.selectorText
+function getSelector(rule: CSSOMRule): string {
+  if (rule.type === CSSRuleTypes.STYLE_RULE) return rule.selectorText
+  if (rule.type === CSSRuleTypes.KEYFRAMES_RULE) {
+    const {name} = rule
+    if (name) return `@keyframes ${name}`
+
+    /**
+     * there is no CSSKeyframesRules.name for browsers like
+     * - IE 9
+     * - Safari 7.1.8
+     * - Mobile Safari 9.0.0
+     */
+    const {cssText} = rule
+    return `@${extractSelector(cssText, cssText.indexOf('keyframes'))}`
```

Comment:
let's use substr/substring, because it's faster https://esbench.com/bench/5930564299634800a034843d